gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/BuildEnvironmentBundleTask.java

```diff
@@ -66,23 +71,38 @@ DirectoryProperty getConfigFolder() {
     public void perform() {
         final EnvironmentBundleCreator environmentBundleCreator = getInstance(EnvironmentBundleCreator.class);
         final List<File> metaDataFiles = collectFiles(into.getAsFile().get().getPath(), YML_EXTENSION);
-        if(metaDataFiles.isEmpty()) {
+        if (metaDataFiles.isEmpty()) {
             throw new MissingEnvironmentException("Metadata file does not exist.");
         }
+
         metaDataFiles.stream().forEach(metaDataFile-> {
             final Pair<String, Map<String, String>> bundleEnvironmentValues = environmentConfigurationUtils.parseBundleMetadata(metaDataFile, configFolder.getAsFile().get());
             if (null != bundleEnvironmentValues) {
-                final String bundleFileName = bundleEnvironmentValues.getLeft() + "." + configName.get() + ".environment.bundle";
+                final String envBundleFilename = getEnvBundleFileName(bundleEnvironmentValues.getLeft());
                 environmentBundleCreator.createEnvironmentBundle(
                         bundleEnvironmentValues.getRight(),
                         into.getAsFile().get().getPath(),
                         into.getAsFile().get().getPath(),
                         EMPTY,
                         PLUGIN,
-                        bundleFileName,
+                        envBundleFilename, // Passing envBundleFilename
                         bundleEnvironmentValues.getLeft()
                 );
             }
         });
     }
+
+    private String getEnvBundleFileName(String deployBundleName) {
+        final String extension = "env" + DocumentFileUtils.INSTALL_BUNDLE_EXTENSION;
```

Comment:
move this extension to DocuemntFileUtils

Reply:
Done